index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <div id="title" title="dr.norman borlaug">
+        <h2>Dr. Norman Borlaug</h2>
+        <p>The man who saved a billion lives</p>
+      </div>
+      <div id="img-div">
+        <img
+          id="image"
+          src="images/tribute-page-main-image.jpg"
+          alt="Image of Mr. Norman Borlaug"
+        />
+        <p id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+      </div>
+      <div id="tribute-info">
+        <h2>Here's a time line of Dr. Borlaug's life:</h2>
+        <ul>
+          <li>1914 - Born in Cresco, Iowa</li>
+          <li>
+            1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to
+            budget cuts. Inspired by Elvin Charles Stakman, he returns to school
+            study under Stakman, who teaches him about breeding pest-resistent
+            plants.
+          </li>
+          <li>1942 - Receives a Ph.D. in Genetics and Plant Pathology</li>
+          <li>
+            1945 - Discovers a way to grown wheat twice each season, doubling
+            wheat yields
+          </li>
+          <li>2009 - dies at the age of 95.</li>
+        </ul>
+      </div>
+      <p id="footer">
+        If you have time, you should read more about this incredible human being
+        on his
+        <a
+          id="tribute-link"
+          href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+          target="_blank"
+          >Wikipedia entry</a
+        >
+      </p>
+    </main>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
- [ ] Good Job for using the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) But I recommend you to **phrased the comments politely, as suggestions instead of orders** and  **always includes an explanation, a "why", for each requested change**. For Example : -
- " Kindly I recommend you to remove the all comments from your code base it makes your code unclear and ambiguous " .